framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1172,10 +1172,17 @@ protected void performFlowAnalysis(ClassTree classTree) {
 
                 while (!lambdaQueue.isEmpty()) {
                     Pair<LambdaExpressionTree, Store> lambdaPair = lambdaQueue.poll();
+                    MethodTree enclosingMethod =
```

Comment:
Possible alternative to think about: change `Queue<Pair<LambdaExpressionTree, Store>> lambdaQueue` to also contain the MethodTree within which the lambda occurred.

Reply:
I am not quite sure which one is better... It seems that we still need to compute the enclosing method if we change `lambdaQueue`. Does the current solution good enough?